apps/deploy_monitoring/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "deploy-monitoring"
-version = "1.17.0"
```

Comment:
@akoptelov `1.16.1` right?

Reply:
Looks like it is not handled by `version.sh`. I'll fix that.

Reply:
Ok, same for the title of the PR (I had exactly the same issue when releasing 1.16.0)

Reply:
Closing this one in favor of https://github.com/tezedge/tezedge/pull/970